app/retail/templates/help.html

```diff
@@ -22,21 +22,21 @@
 <head>
     {% include 'shared/head.html' %}
     {% include 'shared/cards.html' %}
+    <link rel="stylesheet" href="/static/v2/css/help.css" />
```

Comment:
You'll want to update this to `<link rel="stylesheet" href="{% static "v2/css/help.css" %}">`